vagrant_scripts/provision.sh

```diff
@@ -48,13 +48,14 @@ eval "$(pyenv virtualenv-init -)"
 pyenv install 3.6.8
 pyenv virtualenv 3.6.8 swarfarm-3.6.8
 pyenv activate swarfarm-3.6.8
+cd /vagrant
+pyenv local swarfarm-3.6.8
```

Comment:
assuming we want this local to the /vagrant directory, but let me know if you think different

Reply:
I believe it's required to be local to the /vagrant dir